src/pages/questionPage.js

```diff
@@ -64,16 +64,29 @@ const selectAnswer = (event) => {
      selectedListItem.classList.add('yes')
     } else {
      selectedListItem.classList.add('no')
-     showCorrectAnswer()
+     showCorrectAnswer();
+     showInformation();
     }
     
     document
     .getElementById(NEXT_QUESTION_BUTTON_ID)
-    .removeAttribute('disabled')
-  
+    .removeAttribute('disabled') 
 };
 
+// show correct answer and show information
+const showInformation = () => {
+  const answersListElement = document.getElementById(ANSWERS_LIST_ID);
+
+  const questionTexts = quizData.questions.map(question => question.text).join(' ');
+
+  const informationView = document.createElement('div');
```

Comment:
Since you are making a new, separate div here to contain the correct information, could you make the `informationView` its own js file?